test/unit/util/getCssSelectorLocations.js

```diff
@@ -0,0 +1,23 @@
+import test from 'tape';
+
+import getCssSelectorLocations from
+  '../../../src/util/getCssSelectorLocations';
+import {css} from '../../data/acceptance.json';
+
+const [source] = css;
+
+test('finds all selectors in css', (assert) => {
+  const selectors = getCssSelectorLocations(source);
+  assert.deepEqual(selectors.map(s => s.selector), [
```

Comment:
You can do the map a little more cutely with lodash, e.g. `map(selectors, 'selector')`